### PR TITLE
Update R&R column titles

### LINKED ISSUE #4603: Rename R&R columns
## Is your feature request related to a problem? Please describe 👀
A request in from the client:

- Adjustments -> Adjustments +/-
- Consumed -> Consumed/Distributed
- Adjusted Consumption -> Adjusted Consumption/Distribution
- AMC -> AMC/AMD

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #4603

# 👩🏻‍💻 What does this PR do?
Renames the columns as requested.

## 💌 Any notes for the reviewer?

# 🧪 Testing

- [ ] Check the column titles

# 📃 Documentation

- [x] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [ ] **These areas should be updated or checked**: 
  1.
  2.

### CODE REVIEW

File: client/packages/requisitions/src/RnRForms/DetailView/ContentArea.tsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ export const ContentArea = ({
           },
           '.second-column': {
             borderRight: '1px solid blue',
-            left: 79,
+            left: 83,
```
Comment (mark-prins) [inline client/packages/requisitions/src/RnRForms/DetailView/ContentArea.tsx:91]: offroad: fixes a little horizontal scroll jiggle

File: client/packages/common/src/intl/locales/en/replenishment.json [hunk 2]
```diff
@@ -85,11 +85,11 @@
   "label.projected": "Projected",
   "label.quantities": "Quantities",
   "label.requested-packs": "Requested packs",
-  "label.rnr-adjustments": "Adjustments",
-  "label.rnr-consumed": "Consumed",
-  "label.rnr-consumed-adjusted": "Adjusted consumption",
+  "label.rnr-adjustments": "Adjustments +/-",
+  "label.rnr-consumed": "Consumed/distributed",
+  "label.rnr-consumed-adjusted": "Adjusted consumption/distribution",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/replenishment.json:90]: Wondered whether adding a space after the `/`s so they wrap would be nicer?

i.e. this:
Screenshot 2024-08-17 at 12 48 21 AM
vs this:
Screenshot 2024-08-17 at 12 48 28 AM

Not sure which we're more concerned about on tablet - seeing more columns horizontally (as working line by line) or keeping some vertical real estate, especially when keyboard open... up to you 🤷‍♀️

**Review by lache-melvin — APPROVED** [review]
LGTM thanks Mark! Up to you on the wrapping, just thought I'd try it 😁

**Comment by mark-prins** [comment]
thanks - I prefer spacing around words, but took the client requirement directly, which is using all-jammed-up.
have spaced it out 😉 and have also reduced the font, as per Mariya's request.

**Pull request merged**